app/main/presenters/search_presenters.py

```diff
@@ -182,21 +188,24 @@ def build_lots_and_categories_link_tree(framework, lots, category_filter_group,
     Returns a list of selected filters: there will always be at least one element, which is
     the root of the tree; there may then be a category; there may then follow a sub-category.
 
-    :param framework: the latest G-Cloud framework from the API
+    :param framework: the framework from the API
     :param lots: a sequence of lot dicts applicable to the live framework(s)
     :param category_filter_group: a single filter group loaded from the framework search_filters manifest
-    :param request: current request so that we can figure out what lots and categories are selected
-    :param content_manifest: a ContentManifest instance for G-Cloud search_filters.
-    :return: list of selected category and lot filters, starting with the 'all categories' root node node
+    :param request: the original request for creating links for nodes
+    :param cleaned_request_args: current cleaned request args so that we can figure out selected lots and categories
+    :param content_manifest: a ContentManifest instance for the frameworks search_filters.
+    :param search_link_builder: a werkzeug Href object for the view category tree links link to
```

Comment:
Can't work out what this `search_link_builder` does. Could do with a better doc

Reply:
Have updated the doc string with more info.